patterns/1-initial/hivemind-pattern.md

```diff
@@ -0,0 +1,50 @@
+## Title
+
+Hive Mind
+
+## Patlet
+
+TBD
```

Comment:
How about: Provide a safe space and encourage partial/unfinished ideas to change the company culture to encourage and nurture innovation.